server/src/weaverbird/backends/sql_translator/steps/pivot.py

```diff
@@ -35,14 +35,10 @@ def translate_pivot(
         f'query.metadata_manager.query_metadata: {query.metadata_manager.retrieve_query_metadata()}\n'
     )
     aggregate_part = f'{step.agg_function}({step.value_column})'
-    pivot_values = (
-        sql_query_executor(
-            domain=None,
-            query_string=f"""{query.transformed_query} SELECT DISTINCT({step.column_to_pivot}) FROM {query.query_name}""",
-        )
-        .df[f'{step.column_to_pivot}']
-        .values.tolist()
-    )
+    pivot_values = sql_query_executor(
+        domain=None,
+        query_string=f"""{query.transformed_query} SELECT DISTINCT({step.column_to_pivot}) FROM {query.query_name}""",
+    )[f'{step.column_to_pivot}'].values.tolist()
```

Comment:
Du coup on renvoie directement un DataFrame ? Plus un DataSlice ?

Reply:
Je ne suis pas sûr de comprendre ta question. `sql_query_executor` renvoie effectivement un DataFrame (avant c'était annoté `str`, mais je pense que c'était une erreur). Tu penses que ca devrait renvoyer un DataSlice?

Reply:
Ah I see now, you mean this will need an update on Laputa's side as well?
I do agree!

I can't find any reason we'd like a DataSlice here more than just a DataFrame. If you have one in mind, I'm interested